tests/server/tso/manager_test.go

```diff
@@ -239,3 +267,15 @@ func waitAllocatorPriorityCheck(cluster *tests.TestCluster) {
 	// so we sleep longer here to wait.
 	time.Sleep(waitAllocatorPriorityCheckInterval)
 }
+
+func triggerCheckLeader(cluster *tests.TestCluster) {
```

Comment:
I defined this function as a method of `TestCluster` [here](https://github.com/tikv/pd/pull/3365/files#diff-dd6d23125575d84283c08100ec798dfeb1f3e93975d5579da859952bc567b312R565). Maybe we should unify them in this PR?

Reply:
updated.